src/tflite_cpu.cpp

```diff
@@ -397,15 +397,26 @@ class MLModelServiceTFLite : public vsdk::MLModelService,
             state->metadata.inputs.emplace_back(std::move(input_info));
         }
 
-        // As above, but for output tensors.
-        //
         // NOTE: The tflite C API docs state that information about
         // output tensors may not be available until after one round
-        // of inference. We are ignoring that guidance for now per the
-        // unknowns about how metadata will be handled in the future.
+        // of inference. We do a best effort inference on all zero
+        // inputs to try to account for this.
+        for (decltype(num_input_tensors) i = 0; i != num_input_tensors; ++i) {
+            auto* const tensor = TfLiteInterpreterGetInputTensor(state->interpreter.get(), i);
+            const auto tensor_size = TfLiteTensorByteSize(tensor);
+            const std::vector<unsigned char> zero_buffer(tensor_size, 0);
```

Comment:
question - would this work for any data type? I know unsigned char is just like pure bytes, and it's 0, so it seems like it should work all rights